<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MitFxPlcWidget</class>
 <widget class="QWidget" name="MitFxPlcWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>744</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>741</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>PLC配置</string>
   </property>
   <widget class="QComboBox" name="comboBox_form">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>65</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_station_number">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>23</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>25</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>站    号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>68</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>等待时间：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PC_number">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>23</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>35</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PLC型号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>68</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>格   式：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_wait_time">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>67</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Model">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>28</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>25</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PC号：</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>188</y>
     <width>741</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(125, 125, 125);</string>
   </property>
   <property name="text">
    <string>      名称                                                                                                        详细</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>219</y>
     <width>741</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_too_Type">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>139</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>工具：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Add">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>136</y>
     <width>75</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Del">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>136</y>
     <width>75</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Up">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>136</y>
     <width>75</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>上移</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Down">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>136</y>
     <width>75</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>下移</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
